benchmark/src/lib.rs

```diff
@@ -43,6 +43,11 @@ pub async fn run(
         seed: 0,
         repetition_penalty: repetition_penalty.unwrap_or(1.0),
         watermark,
+        use_grammar_constraint: false,
+        grammar: "".to_string(),
+        negative_inputs: "".to_string(),
+        guidance_scale: 1.0,
+        logit_bias: Vec::new(),
```

Comment:
small style change to align with other default initializations in this PR

```suggestion
        grammar: String::new(),
        negative_inputs: String::new(),
        guidance_scale: 1.0,
        logit_bias: Vec::new(),
```